File: src/assets/usage-examples-code/graphics/fill_ellipse/fill_ellipse-1-ellipse-color.cs
using static SplashKitSDK.SplashKit;

OpenWindow("Fill Ellipse", 800, 600);

ClearScreen();
FillEllipse(ColorBlue(), 200, 200, 400, 200);
// Added rectangle with same arguments as above for x, y, width and height
DrawRectangle(ColorRed(), 200, 200, 400, 200);
RefreshScreen();

Delay(4000);

CloseAllWindows();

File: src/assets/usage-examples-code/graphics/fill_ellipse/fill_ellipse-1-ellipse-color.cpp
#include "splashkit.h"

int main()
{
    open_window("Fill Ellipse", 800, 600);

    clear_screen();
    fill_ellipse(COLOR_BLUE, 200, 200, 400, 200);
    // Added rectangle with same arguments as above for x, y, width and height
    draw_rectangle(COLOR_RED, 200, 200, 400, 200);
    refresh_screen();
    
    delay(4000);

    close_all_windows();

    return 0;
}